vectorizers/utils.py

```diff
@@ -312,3 +312,37 @@ def pairwise_gaussian_ground_distance(
             result[j, i] = result[i, j]
 
     return result
+
+
+def procrustes_align(e1, e2):
```

Comment:
Might add a 'anchor' variable to this function in future.  That would allow us to hold one of the embeddings fixed in the case that you had meaningful axes and didn't want to allow re-scaling.

Reply:
Fixed.